src/extensions/default/Brackets-InteractiveLinter/main.js

```diff
@@ -145,46 +149,56 @@ define(function (require, exports, module) {
-     */
-    function appReady() {
-        // Removes the default Brackets JSLint linter
-        CodeInspection.register("javascript", {
-            name: "interactive-linter-remove-jslint",
-            scanFile: $.noop
-        });
-
-        // Load up plugins and wait til they are done loading before we
-        // register any handlers into Brackets
-        pluginManager().done(function(plugins) {
-            for (var iPlugin in plugins) {
-                linterManager.registerLinter(plugins[iPlugin]);
-            }
-
-            EditorManager.on("activeEditorChange.interactive-linter", setDocument);
-            setDocument(null, EditorManager.getActiveEditor());
-
-            // If the linters change, then make sure to rebind the document with the new linter
-            var lastLinters;
-            preferences.on("change", function() {
-                var editor = EditorManager.getActiveEditor();
-                if (!editor) {
-                    return;
-                }
+  function init() {
+      CodeInspection.register("javascript", {
+          name: "interactive-linter-remove-jslint",
+          scanFile: $.noop
+      });
 
-                var language = editor.document.getLanguage().getId();
-                var linters  = preferences.get(language);
+      // Load up plugins and wait til they are done loading before we
+      // register any handlers into Brackets
+      pluginManager().done(function(plugins) {
+          for (var iPlugin in plugins) {
+              linterManager.registerLinter(plugins[iPlugin]);
+          }
 
-                if (linters && !_.isEqual(linters, lastLinters)) {
-                    lastLinters = linters.slice(0);
-                    handleDocumentChange();
+          EditorManager.on("activeEditorChange.interactive-linter", setDocument);
+          setDocument(null, EditorManager.getActiveEditor());
+
+          // If the linters change, then make sure to rebind the document with the new linter
+          var lastLinters;
+          preferences.on("change", function() {
+              var editor = EditorManager.getActiveEditor();
+              if (!editor) {
+                  return;
+              }
+
+              var language = editor.document.getLanguage().getId();
+              var linters = preferences.get(language);
+
+              if (linters && !_.isEqual(linters, lastLinters)) {
+                  lastLinters = linters.slice(0);
+                  handleDocumentChange();
+              }
+          });
+      });
+  }
+
+
+    function appReady() {
+        preferences.definePreference("enabled", "boolean", true);
+        if (preferences.get("enabled")) {
+            init();
+        } else {
+            preferences.on("change", function handleChange(){
+                if(preferences.get("enabled")) {
+                    preferences.off("change", handleChange);
+                    init();
                 }
             });
-
-        });
+          }
```

Comment:
This is wrong.  Look at this:

![screen shot 2017-04-21 at 3 19 16 pm](https://cloud.githubusercontent.com/assets/427398/25292835/fe434adc-26a5-11e7-9cbc-7e68cd4da8c4.png)

See how your `if` and `else` on lines 193 and 195 don't line up with this closing `}`?  They all have to match.